test/JDBC/expected/database_metadata.out

```diff
@@ -149,7 +149,7 @@ master#!#dbo#!#dbmeta_tab2#!#<NULL>#!#<NULL>#!#<NULL>#!#0#!#<NULL>#!#<NULL>#!#<N
 
 dbmeta#!#getMaxConnections
 ~~START~~
-0
+262143
```

Comment:
This could become flaky test because in APG we calculate max_connection using following formula - `LEAST({DBInstanceClassMemory/9531392},5000).`